cadc-download-manager-server/src/main/resources/META-INF/resources/chooser.jsp

```diff
@@ -160,14 +164,27 @@ String bodyFooter = skin + "bodyFooter";
     
     
     <h2>Help</h2>
-    
-    <h3>
-        I want to use the Java option but it didn't work. How can I fix it?
-    </h3>
-    <p>
-        For general help on getting applets or webstart working, we 
-        have a <a href="/JavaTest">Java Test Page</a> with instructions.
-    </p>
+
+    <c:if test="<%=enableWebstart%>" >
+        <h3>
+            I want to use the Java option but it didn't work. How can I fix it?
+        </h3>
+        <p>
+            For general help on getting applets or webstart working, we
+            have a <a href="/JavaTest">Java Test Page</a> with instructions.
+        </p>
+    </c:if>
+    <c:if test="not <%=enableWebstart%>" >
+        <h3>
+            I want to use the Java option but I don't see it on the page. How can I get it?
+        </h3>
+        <p>
+            The download manager configuration file org.opencadc.dlm-server.properties
+            has a value for enabling the webstart option. The value is probably set to false
+            in this file. Setting it to true and reloading the Download Manager should display
+            the webstart option.
+        </p>
+    </c:if>
```

Comment:
The last predicate can be omitted as it is user-facing and exposing the implementation configuration is confusing.